views/layouts/emails.njk

```diff
@@ -5,10 +5,16 @@
         <style>
             @import url('https://use.typekit.net/yyc1dgh.css');
         </style>
-        <link rel="stylesheet" type="text/css" href="/build/latest/stylesheets/emails.css">
+        {% if appData.isDev %}
+            <link rel="stylesheet" type="text/css" href="{{ 'stylesheets/emails.css' | getCachebustedPath }}">
```

Comment:
Yes (we pass through all the `res.locals` and globals in the places where we render nunjucks templates directly). The point of this isn't the cachebust, it's to get a path that works locally (eg. begins with `/assets/build` rather than `/build`) so that when testing locally, the Nunjucks/PDF renderer can find the stylesheets.